### PR TITLE
finished week 5 assignment

### PR DESCRIPTION
**Pull request by erikpark**

### CODE REVIEW

**Comment by wrshoemaker** [comment]
@erikpark could you add the PDF or HTML of the PhyloTraits assignment and resubmit your pull request?

**Comment by erikpark** [comment]
Done, sorry for forgetting!

**Comment by wrshoemaker** [comment]
@erikpark No problem!

File: Week5-Temporal/temporal_assignment.Rmd [hunk 1]
```diff
@@ -110,7 +175,10 @@ a. Does diversity vary among sites? Does this correspond to treatment type?
...
+
 > ***Answer 2b***:
+> When PERMANANOVA was used to test the hypothesis that species abundances vary as a factor of treatment type, we found pretty solid supporting evidence that they do. We see pretty high, significant support (R^2 = .414, p = .003) for our hypothesis that plot type is explaining the variation seen between sites. That said, the R^2 value of .414 isn't amazing, and there is still over half of the variance seen left unexplained by just plot type alone, meaning that there is something else going on with these data that we haven't addressed yet just by looking at plot type.
```
Comment (jaytlennon) [inline Week5-Temporal/temporal_assignment.Rmd]: Good interpretation!

File: Week5-Temporal/temporal_assignment.Rmd [hunk 2]
```diff
@@ -139,8 +261,15 @@ b. What does the ACF function do and how does it relate to the ARMA model? How d
...
+
+> The partial autocorrelation function (PACF) differs from the ACF described above in that it gives information about correlations between two series, after it has already corrected for other correlations that might exist with another lagged series.
+
 > ***Answer 3c***:
+> From the full ARMA model, along with everything else, I have learned here that site 7 abundances have seen two major boom and bust cycles over the years with one falling from ~1985-1990, and another about 8 years later. These cycles are definitely solid trends, yet also contain within them much random noise. These two dramatic cycles dominate much of the timeframe examined here, and seem to make it difficult to get an accurate idea of what an average population level looks like in this site, seemingly complicating the task of future predictions from the ARMA model (which don't seem too good). In general, it is my view that the extremly volatile nature of the populations in plot 7 over the years examined here has lead to a poor outcome from the ARMA model, really limiting its descriptive and predictive power.
```
Comment (jaytlennon) [inline Week5-Temporal/temporal_assignment.Rmd]: From my somewhat limited experience with time series analysis, I think you've hit on something important. Some time series can be more challenging to deal with and interpret than others.

File: Week5-Temporal/temporal_assignment.Rmd [hunk 3]
```diff
@@ -162,8 +379,16 @@ c. What does the result of your F-test tell you?
 d. Of the three RM-ANOVA models with different covariance structures, which one is best? How does this affect the interpretation of your data?  
 
 > ***Answer 4a***:
+> Repeated measures analysis of variance is an ANOVA that specifically works around the standard ANOVA assumption of independence of samples and allows for analysis of samples that have been measured multiple times.
+
```
Comment (jaytlennon) [inline Week5-Temporal/temporal_assignment.Rmd]: importantly, the site or experimental unit is a random factor in a mixed effects model which has important implications for how errors and significance are calculated.

File: Week5-Temporal/temporal_assignment.Rmd [hunk 4]
```diff
@@ -162,8 +379,16 @@ c. What does the result of your F-test tell you?
...
 > ***Answer 4b***:
+> Not particularly. In both control and rodent exclosure plots, there just seems to be a lot of oscillation.
+
 > ***Answer 4c***:
+> If I am intrepreting it correctly, the AR(1) F-test seems to be saying that the plot type is far and away the best explanatory variable for the variation seen between sites. The year and plot type by year interaction terms are both hold very poor explanatory value.
+
+> ***Answer 4d***:
+> I may have done it wrong, because there is very little difference between the three RM-ANOVA models here. The compound and unstructured covariance structures were identical, and not much different than the AR(1) which was the best by a hair. The fact that the AR(1) model is the best, tells me that it is most accurate to assume that an observation at time t is most influenced by observations at time t-1, which seems very logical and straightforward - an easy intrepretation.
```
Comment (jaytlennon) [inline Week5-Temporal/temporal_assignment.Rmd]: How did you compare models? With AIC or BIC? Hard to tell

File: Week5-Temporal/temporal_assignment.Rmd [hunk 5]
```diff
@@ -237,15 +648,25 @@ In the R code chunk below, do the following:
 a. Which plot type has the highest stability in total abundance? How is stability of total abundance measured with the function you learned? How does this measure of stability relate to the coefficient of variation?
 
 b. In your own words, describe the concept of synchrony
+
 c. Interpret the results from the biodiversity-stability relationships you analyzed.
 
 > ***Answer 8a***:
+> The control plot type has the highest stability in total abundance. Here stability was calculated as the inverse of the coefficient of variation, which is the standard deviation of a variable divided by its mean value. 
+
```
Comment (jaytlennon) [inline Week5-Temporal/temporal_assignment.Rmd]: What about inverse Simpsons?

File: Week5-Temporal/temporal_assignment.Rmd [hunk 6]
```diff
@@ -237,15 +648,25 @@ In the R code chunk below, do the following:
...
...challenge facing studies of both of these types of diversity is that of autocorrelation. When looking at different sites spread across a landscape, how can you be sure that any similarity you are seeing is real and not just a function of the close proximity (geographically and environmentally) of the sites? Likewise, when looking at the same site over many years how does one ensure that they are properly removing the effect of sample points looking similar just because they were taken from the same site in successive years and so are similar just because the same things are still there? Autocorrelation, be it spatial or temporal seems like an obvious thing once you know to look for it, but failure to account for it fully can really ruin an otherwise promising study. A related, but slightly different problem in studying these two aspects of diversity is that of ensuring you sample along an adaquate distance. If we know that autocorrelation exists, we also have to know that a good way to combat it and find an actual signal in the data is to make sure there are samples spread out along a long enough distance (geographical or temporal) to seperate out all possible effects of proximity.
```
Comment (jaytlennon) [inline Week5-Temporal/temporal_assignment.Rmd]: Thanks for the thoughtful answer

**Pull request merged**